core/domain/domain.fr.yml

```diff
@@ -184,7 +184,7 @@ responses:
         - title: Non
           payload: "/deny"
 
-  utter_no_symptoms:
+  utter_tested_positive_no_symptoms:
     - text: "Il semble que vous n’ayez pas de symptômes de la Covid-19"
```

Comment:
In English, we have "Although your test is positive, ...". Shouldn't the French message be equivalent?

Reply:
Ah! I forgot to change that one! Good catch!